resources/zib/zib-MedicalDevice.xml

```diff
@@ -206,10 +211,12 @@
       <comment value="There are several zibs that reference zib MedicalDevice on a functional level, but for which the direction of the reference in FHIR is reversed. Consult the mappings on this element for an overview of which zib concepts are mapped to this element. &#xD;&#xA;&#xD;&#xA;Note that some zibs have a reference to MedicalDevice that is further constrained (e.g. with additional constraints on ProductType or AnatomicalLocation), such as HearingFunction and VisualFunction. For these zibs separate DeviceUseStatement profiles have been created (based on this profile) which also include a specific slice on which their respective MedicalDevice concept mapping is present." />
       <alias value="FunctioneleOfMentaleStatus" />
       <alias value="Mobiliteit" />
+      <alias value="Darmfunctie" />
       <type>
         <code value="Reference" />
         <targetProfile value="http://nictiz.nl/fhir/StructureDefinition/zib-FunctionalOrMentalStatus" />
         <targetProfile value="http://nictiz.nl/fhir/StructureDefinition/zib-Mobility" />
+        <targetProfile value="http://nictiz.nl/fhir/StructureDefinition/zib-BowelFunction" />

```

Comment:
The nl-core variant of this targetProfile should be added to nl-core-MedicalDevice.

Reply:
Done